lore/settings.py

```diff
@@ -189,6 +189,7 @@ def get_var(name, default):
 )
 COMPRESS_PRECOMPILERS = (
     ('text/requirejs', 'requirejs.RequireJSCompiler'),
+    ('text/jsx', 'node_modules/.bin/jsx < {infile} > {outfile}')
```

Comment:
You will need to add the local installation of node modules to the Dockerfile, possibly replacing the existing -g option used currently, also, heroku currently isn't installing node packages since we are running off the Python buildpack, so we will need to switch to multipack and include node before we merge this.